.github/workflows/on_push_pr.yaml

```diff
@@ -7,70 +7,22 @@ on:
       - master
       - renovate/**
   pull_request:
+  workflow_dispatch:
 
 env:
-  TAG: "v0.0.0" # needed for goreleaser windows builds
-  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
-  ORIGINAL_REPO_NAME: "newrelic/nri-kafka"
+  ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
```

Comment:
I think this is needed for the test job, if so I would move the env to that job so is clear that the other one doesn't need this

Reply:
done!